src/components/P/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+interface Props {
+  fontSize?: string;
+  fontWeight?: string;
+  color?: string;
+  textAlign?: string;
+  marginLeft?: string;
+  marginBottom?: string;
+  marginTop?: string;
+  width?: string;
+  shadow?: string;
+}
+
+const P = styled.p<Props>`
```

Comment:
You don't use it anywhere right now, right?

Reply:
Yes, at first, we thought we could use it within the Logo component (for the text), but then we changed our minds and imported the whole logo.